<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RXDateSelectPage</class>
 <widget class="QWidget" name="RXDateSelectPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>181</width>
    <height>147</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>RXDateSelectPage</string>
  </property>
  <property name="styleSheet">
   <string notr="true">outline:none;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6" stretch="6,2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0px;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="data_show_mode" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(30,30,30);</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,5,5,5,5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="year_mode" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="last_year">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>2018</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="cur_year">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 255, 255);
background-color: rgb(50, 50, 50);
</string>
              </property>
              <property name="text">
               <string>2019</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="next_year">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>2020</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(150, 150, 150);</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,1,1" rowminimumheight="1,1,1">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="horizontalSpacing">
             <number>2</number>
            </property>
            <property name="verticalSpacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QWidget" name="widget_4" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(30, 30, 30);</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QWidget" name="widget_7" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(30, 30, 30);</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QWidget" name="widget_5" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(50, 50, 50);</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QWidget" name="widget_8" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(50, 50, 50);</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QWidget" name="widget_6" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(30, 30, 30);</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QWidget" name="widget_9" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(30, 30, 30);</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="month_mode" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="last_month">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>01</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="cur_month">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(50, 50, 50);
color: rgb(0, 255, 255);
</string>
              </property>
              <property name="text">
               <string>02</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="next_month">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>03</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(150, 150, 150);</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,1,1" rowminimumheight="1,1,1">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="horizontalSpacing">
             <number>2</number>
            </property>
            <property name="verticalSpacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <widget class="QWidget" name="widget_10" native="true">
              <property name="minimumSize">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(30, 30, 30);</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QWidget" name="widget_11" native="true">
              <property name="minimumSize">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(30, 30, 30);</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QWidget" name="widget_12" native="true">
              <property name="minimumSize">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(50, 50, 50);</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QWidget" name="widget_13" native="true">
              <property name="minimumSize">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(50, 50, 50);</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QWidget" name="widget_14" native="true">
              <property name="minimumSize">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(30, 30, 30);</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QWidget" name="widget_16" native="true">
              <property name="minimumSize">
               <size>
                <width>1</width>
                <height>1</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(30, 30, 30);</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="day_mode" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QVBoxLayout" name="day_mode_layout" stretch="1,1,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="last_day">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>01</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="cur_day">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 255, 255);
background-color: rgb(50, 50, 50);</string>
              </property>
              <property name="text">
               <string>02</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="next_day">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>03</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="btn_group" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(30,30,30);</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="2,4,2" columnstretch="5,7,5,7,5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>9</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>9</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="date_confirm_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>1</width>
          <height>1</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>12312</width>
          <height>11111</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(50, 50, 50);</string>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>51</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="cancel_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>1</width>
          <height>1</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>111111</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(50, 50, 50);</string>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="3">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
